scipy/sparse/linalg/dsolve/SuperLU/SRC/msc_stdint.h

```diff
@@ -33,6 +33,11 @@
 #error "Use this header only with Microsoft Visual C++ compilers!"
 #endif // _MSC_VER ]
 
+#if _MSC_VER >= 1900
```

Comment:
Why? It's not going to be updated because it only applies to previous compiler versions.